pkg/rules/rules.go

```diff
@@ -143,12 +144,9 @@ func dedupRules(rules []*rulespb.Rule, replicaLabels map[string]struct{}) []*rul
 		return rules
 	}
 
-	// Remove replica labels and sort each rule's label names such that they are comparable.
+	// Remove replica labels
 	for _, r := range rules {
 		removeReplicaLabels(r, replicaLabels)
-		sort.Slice(r.GetLabels(), func(i, j int) bool {
-			return r.GetLabels()[i].Name < r.GetLabels()[j].Name
-		})
```

Comment:
Is it because labels builder produces sorted labels?

Reply:
Yeah